driven/storage/adapter.go

```diff
@@ -158,36 +160,29 @@ func (sa *Adapter) cacheServiceRegs() error {
 	return nil
 }
 
-func (sa *Adapter) setCachedServiceRegs(serviceRegs *[]model.ServiceReg) {
+func (sa *Adapter) setCachedServiceRegs(serviceRegs *[]model.ServiceRegistration) {
```

Comment:
Should we update function names to match the change to the model and collection names for consistency?

Reply:
Since the APIs will still call these `service-reg` for now I think we can leave it as is